core/arch/aarch64/proc.c

```diff
@@ -71,13 +71,15 @@ DR_API
 int
 proc_num_simd_saved(void)
 {
+    SELF_UNPROTECT_DATASEC(DATASEC_RARELY_PROT);
```

Comment:
?  This is a read.  It should not unprotect .data.

Reply:
Argh. Wrong function, fixed.

Reply:
But it was on the wrong function in 3 different places!  I would expect sthg like that to jump out when looking over pre-PTAL.